### PR TITLE
zabbix_template: Add support for template tags

### PR DESCRIPTION
**Pull request by mu1f407**
##### SUMMARY
Adds support for template tags into zabbix_template module.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
zabbix_template

### CODE REVIEW

File: plugins/modules/zabbix_template.py [hunk 1]
```diff
@@ -429,9 +462,17 @@ def check_template_changed(self, template_ids, template_groups, link_templates,
             if template_macros != existing_macros:
                 changed = True
 
+        if LooseVersion(self._zbx_api_version) >= LooseVersion('4.2'):
+            if 'tags' not in existing_template['zabbix_export']['templates'][0]:
+                existing_template['zabbix_export']['templates'][0]['tags'] = []
+            if template_tags is not None:
+                existing_tags = existing_template['zabbix_export']['templates'][0]['tags']
+                if template_tags != existing_tags:
```
Comment (D3DeFi) [inline plugins/modules/zabbix_template.py:470]: I wonder if we should not rather utilize helper function for comparing list of dictionaries from module_utils/

**Review by D3DeFi — APPROVED** [review]
Very nnice addition, thank you

**Comment by D3DeFi** [comment]
Failing checks may be because you edited template1_50_lower.json, which is also used for zabbix 4.0 I think

**Comment by mu1f407** [comment]
> Failing checks may be because you edited template1_50_lower.json, which is also used for zabbix 4.0 I think

Previously it failed with another issue, but later it failed exactly on this. I've reverted this change and use different approach in tests.

File: plugins/modules/zabbix_template.py (hunk 1 above)
Comment (mu1f407) [reply plugins/modules/zabbix_template.py:470]: I agree, will try it later. Initially I've used the same handling as for macros.

Comment (mu1f407) [reply plugins/modules/zabbix_template.py:470]: Well, I tried to use helpers, but the code was less readable IMO and the result was the same. If its working fine using just simple compare, than I would leave it as it. I think helpers can be useful for some more complex cases.

Comment (D3DeFi) [reply plugins/modules/zabbix_template.py:470]: ok, in that case I have no further objections :)

**Review by D3DeFi — APPROVED** [review]
LGTM

**Pull request merged**